### PR TITLE
Update uploadDockerImage.yml

### PR DESCRIPTION
**Pull request by f-meloni**
I'm assuming `DOCKER_GITHUB_PASSWORD` is actually a token with write permissions for the Packages of this repo

### CODE REVIEW

File: .github/workflows/uploadDockerImage.yml [hunk 1]
```diff
@@ -3,17 +3,28 @@ name: Upload tag to GitHub
...
+      run: echo $PACKAGES_WRITE_TOKEN | docker login docker.pkg.github.com -u $USERNAME --password-stdin
+      env:
+        PACKAGES_WRITE_TOKEN: ${{ secrets.DOCKER_GITHUB_PASSWORD }}
+        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
+    - name: Docker Build
+      run: docker build -t docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION .
+      env:
+        USERNAME: ${{ secrets.DOCKER_GITHUB_USERNAME }}
+    - name: Deploy
+      run: docker push docker.pkg.github.com/danger/danger-js/danger-danger-js:$VERSION
```
Comment (devnoname120) [inline .github/workflows/uploadDockerImage.yml:28]: Requires auth to download (even public) images. Why not use Docker's registry instead?

**Comment by danger-public** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  Please add a changelog entry for your changes. You can find it in `CHANGELOG.md` 

Please add your change and name to the master section.
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 968d9de318ed0995214ed68914bd585c5054c5cb
</p>

File: .github/workflows/uploadDockerImage.yml (hunk 1 above)
Comment (f-meloni) [reply .github/workflows/uploadDockerImage.yml:28]: Yes, I know, I highlighted this in https://github.com/danger/danger-js/pull/810#issuecomment-618001174
But the credentials we have are for GitHub package registry, then I'm fixing the current script, if use or not dockerhub is a separate discussion, and can be done in parallel.

**Comment by f-meloni** [comment]
@orta are you ok with this? If yes I will update the changelog and merge it

**Comment by orta** [comment]
Seems reasonable to me 👍

**Pull request merged**